cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -330,28 +330,6 @@ class HydratorPlusPlusTopPanelCtrl {
     }
   }
 
-  updateTimerLabelAndTitle(res) {
```

Comment:
Is this removal correct?

Reply:
Yes - the function was defined (identically) twice, probably as the result of an incorrect merge conflict in a previous PR. You can see the first definition directly above the removed code.